designer/server/src/routes/forms/create.js

```diff
@@ -34,13 +41,31 @@ const schema = Joi.object().keys({
   })
 })
 
+/**
+ * @type {FailAction}
+ */
+export const displayJoiFailures = (request, h, error) => {
```

Comment:
We tend to export functions so the `displayJoiFailures` name appears in stack traces

Rather than the anonymous array function

```suggestion
export function displayJoiFailures(request, h, error) {
```

Reply:
Could rename it to `redirectToStepWithErrors()` and include the "which has the error?" logic from the final fail action if that's helpful? These checks etc:

```js
// Optionally redirect to errors on previous steps
if ('title' in formErrors) {
  redirectTo = '/create/title'
} else if ('organisation' in formErrors) {
  redirectTo = '/create/organisation'
}
```

We only put it at the end to avoid duplication but don't need to now

Reply:
Done, consolidated.